platform/kobo/koreader.sh

```diff
@@ -17,31 +17,37 @@ if [ "${SCRIPT_DIR}" != "/tmp" ]; then
 fi
 
 # Attempt to switch to a sensible CPUFreq governor when that's not already the case...
-IFS= read -r current_cpufreq_gov <"/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor"
+# Swap every CPU at once if available
+if [ -d "/sys/devices/system/cpu/cpufreq/policy0" ]; then
+    CPUFREQ_SYSFS_PATH="/sys/devices/system/cpu/cpufreq/policy0"
+else
+    CPUFREQ_SYSFS_PATH="/sys/devices/system/cpu/cpu0/cpufreq"
+fi
```

Comment:
How does this swap **every** CPU? Isn't this only changing policy for CPU0?

Reply:
Policy is global (IIRC).

Reply:
It actually doesn't really matter, because...

```
┌─(ROOT@cadmus:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(0.28:28%:16:45:99%:#)── ls -lash /sys/devices/system/cpu/cpu0/cpufreq                                                                                                                                                                                                                                                                               ──(Sun, Jan 16)─┘
     0 lrwxrwxrwx    1 admin    root           0 Jan 16 16:44 /sys/devices/system/cpu/cpu0/cpufreq -> ../cpufreq/policy0
┌─(ROOT@cadmus:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(0.22:28%:16:45:99%:#)── ls -lash /sys/devices/system/cpu/cpu1/cpufreq                                                                                                                                                                                                                                                                               ──(Sun, Jan 16)─┘
     0 lrwxrwxrwx    1 admin    root           0 Jan 16 16:44 /sys/devices/system/cpu/cpu1/cpufreq -> ../cpufreq/policy0
┌─(ROOT@cadmus:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(0.20:28%:16:45:99%:#)── ls -lash /sys/devices/system/cpu/cpu2/cpufreq                                                                                                                                                                                                                                                                               ──(Sun, Jan 16)─┘
     0 lrwxrwxrwx    1 admin    root           0 Jan 16 16:45 /sys/devices/system/cpu/cpu2/cpufreq -> ../cpufreq/policy0
┌─(ROOT@cadmus:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(0.18:28%:16:45:99%:#)── ls -lash /sys/devices/system/cpu/cpu3/cpufreq                                                                                                                                                                                                                                                                               ──(Sun, Jan 16)─┘
     0 lrwxrwxrwx    1 admin    root           0 Jan 16 16:45 /sys/devices/system/cpu/cpu3/cpufreq -> ../cpufreq/policy0
```

;o).